### PR TITLE
feat(console): add placeholders

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add placeholder to inputs according to: https://www.notion.so/silverhand/Text-Field-Placeholder-text-a5ba4c1578f34c8f8761497d359ee416

## Testing

Local tested, part of screenshots:

截屏2022-06-28 下午4 53 54
截屏2022-06-28 下午4 53 56

### CODE REVIEW

File: packages/phrases/src/locales/en.ts [hunk 1]
```diff
@@ -117,11 +120,16 @@ const translation = {
...
+      application_name_placeholder: 'My App',
       description: 'Description',
+      description_placeholder: 'Enter your application description',
       authorization_endpoint: 'Authorization endpoint',
       redirect_uri: 'Redirect URIs',
+      redirect_uri_placeholder: 'https://myapp.com/login',
       post_sign_out_redirect_uri: 'Post Sign-out Redirect URIs',
+      post_sign_out_redirect_uri_placeholder: 'https://myapp.com/login',
       cors_allowed_origins: 'CORS allowed origins',
+      cors_allowed_origins_placeholder: 'https://login.mydomain.com',
```
Comment (gao-sun) [inline packages/phrases/src/locales/en.ts:132]: sign-in?

File: packages/phrases/src/locales/en.ts [hunk 2]
```diff
@@ -261,6 +272,7 @@ const translation = {
       field_username: 'Username',
       field_name: 'Name',
       field_avatar: 'Avatar image URL',
+      field_avatar_placeholder: 'https://your-avatar/',
```
Comment (gao-sun) [inline packages/phrases/src/locales/en.ts:275]:
```suggestion
      field_avatar_placeholder: 'https://your.cdn.domain/avatar.png',
```

File: packages/phrases/src/locales/en.ts [hunk 3]
```diff
@@ -325,7 +337,9 @@ const translation = {
           logo: 'App logo only',
         },
         logo_image_url: 'App logo image URL',
+        logo_image_url_placeholder: 'https://your-cdn-domain/logo.png',
         dark_logo_image_url: 'App logo image URL (Dark)',
+        dark_logo_image_url_placeholder: 'https://your-cdn-domain/logo-dark.png',
```
Comment (gao-sun) [inline packages/phrases/src/locales/en.ts:342]: ditto

**Review by simeng-li — APPROVED** [review]
overall LGTM, please address other comments accordingly.

**Pull request merged**